.travis.yml

```diff
@@ -8,11 +8,15 @@ rvm:
   - 2.2
   - 2.3.3
   - 2.4.0
+  - ruby-head
 
 matrix:
   include:
     - rvm: 2.3.0
       script: bundle exec rake test_isolated
+  allow_failures:
+    - rvm: ruby-head
+  fast_finish: true
```

Comment:
@rafaelfranca we do have those in Rails.

(I'm pretty skeptical that it's worth the CPU to include ruby-head at all, tbh... no-one seems to pay much attention until December.)